Source/Core/Core/HW/GCMemcard/GCMemcard.cpp

```diff
@@ -50,15 +50,6 @@ GCMemcard::GCMemcard(const std::string& filename, bool forceCreation, bool shift
   }
   else
   {
-    // This function can be removed once more about hdr is known and we can check for a valid header
```

Comment:
Shouldn't we actually implement this valid header check?

Reply:
I suppose so. In this PR or in a separate one?

Reply:
I'd do it here (by replacing this filter) if we have an easily usable method for it (or something that can easily be turned into such a method); otherwise in a follow-up PR (to make reviews easier)

Reply:
Apart from the Dir/BAT checksum calculations that are right below this block and the rudimentary filesize checks I'm not seeing any existing code that checks for card validity.

Reply:
Different PR it is then.

Reply:
Followup at #8020.